### PR TITLE
Homework Profiling

### PR DESCRIPTION
**Pull request by UsernameAlexei**

### CODE REVIEW

File: Profiling_task.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    for i in range(len(arr)):
+        minimum = i
+
+        for j in range(i + 1, len(arr)):
+            if arr[j] < arr[minimum]:
+                minimum = j
+
+        arr[minimum], arr[i] = arr[i], arr[minimum]
+
+    return arr
```
Comment (arybin93) [inline Profiling_task.py:68]: Could you please add implementation by using `sorted` method or `sort` method for the list to your analysis?
Using `pandas` methods for sorting also interesting.

File: Profiling_task.py [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+
+# 4 unpacking a nested list way 1
+def unpacking_1(un_list):
+    up_1 = " ".join(" ".join(L) for L in un_list)
```
Comment (arybin93) [inline Profiling_task.py]: It will work only for a list with string values? How about a more general approach?

File: Profiling_test_task.py [hunk 3]
```diff
@@ -0,0 +1,117 @@
+# TEST TASK
+import unittest
```
Comment (arybin93) [inline Profiling_test_task.py]: Try to use pytest as well, you will like it

File: Profiling_task_memory output.txt [hunk 4]
```diff
@@ -0,0 +1,125 @@
+C:\ProgramData\Anaconda3\python.exe "C:/Users/Алексей/Desktop/EPAM/EPAM Python stage 2/Profiling/Profiling_task_memory.py"
```
Comment (arybin93) [inline Profiling_task_memory output.txt]: Please provide an analysis of your results. Try to explain why we got these results. Which implementation is best and why?

**Review by arybin93 — CHANGES_REQUESTED** [review]
Thank you for your work. I've left comments, please consider and resolve them

**Review by arybin93 — APPROVED** [review]
Good work. Accepted. 👍

**Comment by UsernameAlexei** [comment]
Thank you

**Pull request merged**